tests/unit/Amalgamation.spec.ts

```diff
@@ -3,58 +3,56 @@ import Vuetify from 'vuetify'
 import { mount } from '@vue/test-utils'
 import { createPinia, setActivePinia } from 'pinia'
 import { useBusinessStore } from '@/stores'
-import Amalgamation from '@/components/Dashboard/Alerts/Amalgamation.vue'
+import Alerts from '@/components/Dashboard/Alerts.vue'
 import { ContactInfo } from '@/components/common'
-import flushPromises from 'flush-promises'
 
 Vue.use(Vuetify)
 const vuetify = new Vuetify({})
 
 setActivePinia(createPinia())
 const businessStore = useBusinessStore()
 
-describe('Amalgamation component', () => {
+describe('Amalgamation alert', () => {
   beforeAll(() => {
+    businessStore.$state.businessInfo.goodStanding = true
     businessStore.$state.businessInfo.warnings = [
       {
-        code: 'AMALGAMATING_BUSINESS',
-        message: 'This business is part of a future effective amalgamation.',
+        code: null,
+        message: null,
         warningType: 'FUTURE_EFFECTIVE_AMALGAMATION',
-        data: {
-          amalgamationDate: '2024-01-31T08:00:00+00:00'
-        }
+        data: { amalgamationDate: '2024-01-31T08:00:00+00:00' }
       }
     ]
   })
 
   it('Displays expansion panel closed', () => {
-    const wrapper = mount(Amalgamation, { vuetify })
+    const wrapper = mount(Alerts, { vuetify })
```

Comment:
I tried mounting just the Amalgamation component, but vitest complained that an expansion panel should be nested in a list of panels, and I couldn't find any documentation on how to mount one component inside another one.

Instead, I mount the Alerts component. Because of the state of the store (above), just the Amalgamation alert should be displayed, and I verify that below.